src/components/DataDrawer/Chart/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+import colormap from 'colormap';
+import { shuffle } from 'lodash';
+import { ChartConfig } from '../../../config/types';
+import { TableData } from '../../../context/tableStateSlice';
+
+type ChartProps = {
+  title: string;
+  data: TableData;
+  config: ChartConfig;
+};
+
+export function Chart({ title, data, config }: ChartProps) {
+  const header = data.rows[0];
+  const tableRows = data.rows.slice(1, data.rows.length);
+  const colors = shuffle(
+    colormap({
+      colormap: 'rainbow-soft',
+      nshades: tableRows.length,
+      format: 'hex',
+      alpha: 0.7,
+    }),
+  );
+  try {
+    const indices = Object.keys(header).filter(key =>
+      key.includes(config.xAxis || ''),
+    );
+    const labels = indices.map(index => header[index]);
+    const datasets = tableRows.map((row, i) => ({
+      label: (row[config.category] as string) || '',
+      fill: true,
+      backgroundColor: colors[i],
+      borderWidth: 2,
+      data: indices.map(index => row[index] as number),
```

Comment:
This cast has the potential to be a little "dangerous", from a TS perspective (basically losing the type protections that TS gives you and reverting to the possibility of a runtime error in JS). There's nothing that guarantees that `row[index]` actually is a `number`... (it could probably be a `string` or `undefined` as well)